server/commands/roleplay.py

```diff
@@ -149,7 +149,7 @@ def ooc_cmd_roll(client, arg):
     """
     roll, num_dice, chosen_max, modifiers, Sum = rtd(arg)
 
-    client.area.broadcast_ooc(f'{client.showname} rolled {roll} out of {chosen_max}.\nThe total sum is {Sum}.')
+    client.area.broadcast_ooc(f'{client.showname} rolled {roll} out of {chosen_max}.' + f'\nThe total sum is {Sum}.' if num_dice > 1 else '')
```

Comment:
Had to encase this line in () after the + sign, otherwise it interpreted the whole part before "if" as what to display if num_dice is more than >